Add onboarding example

beth-panx opened the pull request:
### PR Type
- Feature

### Description of the changes
Including onboarding sample from Ignite

shweaver-MSFT reviewed (changes requested):
A few notes:
1. Looks good when signed in, but otherwise looks odd when signed out. Perhaps hide content when not signed in, or add a "Please sign in" message.
1. Page is not responsive. Sections should stack when the view is shrunk.
1. I can't sign in! At least not straight on the page. Likely to do with the redirect uri. I had to sign in and cache my creds on the index.html page before navigating to the sample.

![image](https://user-images.githubusercontent.com/25832310/93142635-466af580-f69b-11ea-9d48-65f473482d8f.png)
![image](https://user-images.githubusercontent.com/25832310/93142856-a95c8c80-f69b-11ea-85c9-03271d84559b.png)

shweaver-MSFT commented:
When I sign in, I see scroll bars for each section. We should probably only have the one on the right.

![image](https://user-images.githubusercontent.com/25832310/93143554-086ed100-f69d-11ea-8f32-fbe2cbbf4d21.png)

beth-panx commented:
> When I sign in, I see scroll bars for each section. We should probably only have the one on the right.
> 
> ![image](https://user-images.githubusercontent.com/25832310/93143554-086ed100-f69d-11ea-8f32-fbe2cbbf4d21.png)

This is based on design Roman provided. That each section will scroll.

beth-panx commented:
@shweaver-MSFT I finally made some updates! 🙈 Thanks for reviewing!!
1. Updated some simple logic to hide/show base on sign in state. This is very basic css 'display: none' class addition. I didn't add too much code to handle this more gracefully since this is not the key demonstration point for the demo.
2. Updated responsive design to have full screen components once screen gets narrower. 
3. I am not sure what the sign in problem is for you... I am using the same client-id as our index.html one.
